PKGBUILD

```diff
@@ -0,0 +1,20 @@
+# Maintainer: Julius de Jeu <julius@voidcorp.nl>
+pkgname=cadmium
+pkgver=0.1.0
+pkgrel=1
+pkgdesc="A simple Display Manager"
+makedepends=('rust' 'cargo')
+depends=('pam' 'libxcb' 'dbus')
+arch=('i686' 'x86_64' 'armv6h' 'armv7h')
+url=https://github.com/jonay2000/cadmium
+license=('wtfpl')
```

Comment:
This is a good incentive to decide on a License, GPL or MIT probably?
@jonay2000

Reply:
we decided on MIT